### DISCUSSION TITLE
https://eng06.sinotech-eng.com/v2/zh/guide/

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#840] [Announcements]
# https://eng06.sinotech-eng.com/v2/zh/guide/

介绍 VuePress Theme Hope 是一个包含大量功能的 VuePress 主题。 它被设计为一个强大且可扩展的主题，内置了大量主题插件。 首先，您应该已经对以下内容有基本的了解： （包括 概念） 提示 如果您对 VuePress、Markdown 或其他东西完全陌生，请从我们的 开始。 如果您熟悉上述内容，请。 一个拥有插件和强大主题的项目...

https://eng06.sinotech-eng.com/v2/zh/guide/

**Comment by sinotec2** [comment]
這是一個測試
